komodo-relational/src/main/java/org/komodo/relational/teiid/Teiid.java

```diff
@@ -257,4 +259,44 @@ public Teiid resolve( final UnitOfWork transaction,
      * @throws KException
      */
     CachedTeiid importContent(UnitOfWork transaction) throws KException;
+
+    /**
+     * Updates the teiid cache with the specified server VDB.  If cache object with same name exists, it is replaced.
+     * @param uow
+     *        the transaction (cannot be <code>null</code> or have a state that is not {@link State#NOT_STARTED})
+     * @param vdbName 
+     *        the VDB name
+     * @throws KException
+     */
+    void updateCacheWithServerVdb(UnitOfWork uow, String vdbName) throws KException;
```

Comment:
Wouldn't it make more sense to ask the cache to update its vdb, driver, datasource collections? This seems prone to updating the cache rather than asking for an update from the server. I realise the implementations may ask the server but the method signatures imply otherwise.